### PR TITLE
Multicast KNoT Web IP and port

### LINKED ISSUE #7: Broadcast IP and back-end Port through mDNS
As a developer, I'd like to expose my local IP and Port through multicast DNS allowing mobile clients to easily fetch the route to access the web UI backend to make requests about connected things or Open Thread configuration.

### PR DESCRIPTION
**Pull request by alla3**
Add knot-web.service on Avahi to expose the gateway IP and Port through multicast DNS, allowing mobile clients such as Android Setup App to get this information and establish a connection with Web UI.

Resolves #7

### CODE REVIEW

**Review by jcorde — CHANGES_REQUESTED** [review]
Remove IPV6 support **_so_** Android Setup App **_uses_** a stable IPV4 connection.

**Review by jcorde — CHANGES_REQUESTED** [review]
Please, mention the service that is being included on the title. 
_"a new service"_ is too generic. Something like _" Add local address multicasting to Avahi"_ might be more fit.

File: package/avahi/avahi.mk [hunk 1]
```diff
@@ -197,6 +197,13 @@ define AVAHI_INSTALL_SYSTEMD_SYSUSERS
 endef
 endif
 
+define AVAHI_KNOT_SERVICE
```
Comment (ramonhpr) [inline package/avahi/avahi.mk]: Shouldn't we add this service to another package related to KNoT? This way we will always import this service independent if we build a KNoT image or not. 
I suggest we move this to the knot-web package as it's exposing its HTTP port. What do you think?
If you agree, remember to make avahi package a dependency from the knot-web package.

File: package/avahi/avahi.mk [hunk 2]
```diff
@@ -204,6 +204,13 @@ endef
 
 AVAHI_POST_INSTALL_TARGET_HOOKS += AVAHI_KNOT_SERVICE
 
+define AVAHI_DISABLE_IPV6
```
Comment (ramonhpr) [inline package/avahi/avahi.mk:201]: You are going to disable IPv6 always doing that. 
Here you should create a flag to enable/disable IPv6 in ` package/avahi/Config.in` file. As the default is to enable ipv6 you should also add the flag disabling it on `configs/knot_gateway_defconfig`
Like this:
`ifeq ($(BR2_PACKAGE_AVAHI_DISABLE_IPV6),y)`
`... your changes`
`endif`

**Review by ramonhpr — CHANGES_REQUESTED** [review]

File: package/avahi/avahi.mk (hunk 1 above)
Comment (alla3) [reply package/avahi/avahi.mk]: Indeed. I have made changes to my previous version.

**Review by ramonhpr — APPROVED** [review]

**Review by jcorde — APPROVED** [review]
LGTM

**Pull request merged**